src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -3,10 +3,10 @@
 Release History
 ===============
 
-* Remove erroneous print statement for `az webapp auth update`
-
 0.2.14
 ++++++
+* webapp, functionapp: az webapp/functionapp deployment list-publishing-credentials, get the Kudu (scm) url and its credentials
```

Comment:
you should move this to be above the `0.2.14`. 0.2.14 has been released.

Reply:
done.